workers/unity/Assets/Playground/Scripts/UI/ConnectionScreenController.cs

```diff
@@ -0,0 +1,105 @@
+using Improbable.Gdk.Core;
+using UnityEngine;
+
+namespace Playground
+{
+    [ExecuteInEditMode]
+    public class ConnectionScreenController : MonoBehaviour
+    {
+        private const string HostIpPlayerPrefsKey = "SpatialOSHostIp";
+        private const int GuiPadding = 10;
+        private const float VerticalPositionRatio = 0.4f;
+
+        [SerializeField] private GameObject clientWorkerPrefab;
+        [SerializeField] private Font font;
+        [SerializeField] private float screenWidthFontRatio = 20;
+
+        private bool isConnecting;
+        private string ipAddressText;
+        private string errorMessage = string.Empty;
+
+        private GameObject worker;
+
+        public void Awake()
+        {
+            ipAddressText = PlayerPrefs.GetString(HostIpPlayerPrefsKey);
```

Comment:
returns the default value, so an empty string: 
https://docs.unity3d.com/ScriptReference/PlayerPrefs.GetString.html

Reply:
That's if you use the `public static string GetString(string key, string defaultValue);` - it doesn't say what happens if you use `public static string GetString(string key);`? If its truly a default string (as in `default(string)`), it will return null I think

Reply:
looked at the decompiled code :P
```    
public static string GetString(string key)
{
    return PlayerPrefs.GetString(key, "");
}
```

it does return an empty string

Reply:
Fair enough 😛